src/components/common/Footer/LinksList.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react'
+import {
+  Link,
+  Icon,
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionPanel,
+} from '@faststore/ui'
+
+import { PlusButtonIcon } from './Icons'
+
+function LinksList() {
```

Comment:
Maybe we can just call it `FooterLinks`

Reply:
Great suggestion! I'm going to change it.